packages/driver/src/cypress/state.ts

```diff
@@ -0,0 +1,35 @@
+/// <reference path="../../types/cy/commands/session.d.ts" />
```

Comment:
New file is created because types defined in `.ts` cannot be imported to `d.ts` files without breaking `d.ts`'s auto import feature.